### PR TITLE
Updates to the Rate Limit SMS Endpoint List

### PR DESCRIPTION
**Pull request by christinechoy-okta**
## Description:
- **What's changed?** Added endpoints to the SMS and Call rate limits section
- **Is this PR related to a Monolith release?** No

### Resolves:

* [OKTA-428184](https://oktainc.atlassian.net/browse/OKTA-428184)

### CODE REVIEW

**Review by brentschaus-okta — APPROVED** [review]
LGTM!

File: packages/@okta/vuepress-site/docs/reference/rl-additional-limits/index.md [hunk 1]
```diff
@@ -101,11 +101,14 @@ API endpoints that take username and password credentials, including the [Authen
   **Endpoints**
   * `/api/v1/authn/factors`
   * `/api/v1/users/${userId}/factors`
+  * `/user/profile/mobile_phone`
```
Comment (haoyuli-okta) [inline packages/@okta/vuepress-site/docs/reference/rl-additional-limits/index.md:104]: This is not a public endpoint for customers, even though we rate limit it, but it is triggered through V1 UI

Comment (christinechoy-okta) [reply packages/@okta/vuepress-site/docs/reference/rl-additional-limits/index.md:104]: I removed this endpoint.

**Review by haoyuli-okta — APPROVED** [review]
LGTM

**Comment by shayekhazaeli-okta** [comment]
@christinechoy-okta `/idx` endpoints are not publicly documented, are we adding the rate limit info for endpoints that are not publicly available?
cc @damianali-okta

**Pull request merged**